src/Worker/Contact/Remove.php

```diff
@@ -29,19 +29,22 @@
  */
 class Remove extends RemoveContent
 {
-	public static function execute(int $id): array
+	public static function execute(int $id): bool
 	{
-		$contact = parent::execute($id);
+		// Only delete if the contact is to be deleted
+		$contact = DBA::selectFirst('contact', ['id', 'uid', 'url', 'nick', 'name'], ['deleted' => true, 'id' => $id]);
+		if (!DBA::isResult($contact)) {
+			return false;
+		}
 
-		if (!empty($contact)) {
-			return [];
+		$result = parent::execute($id);
+		if (!empty($result)) {
```

Comment:
`$result` can only be bool, can't it? So checking with `empty` seems to be unneeded, I think.

Reply:
Yeah, it used to be an array in a previous iteration.